OpenSim/Tools/Test/vis_BuiltinGeometry.txt

```diff
@@ -118,9 +118,3 @@ DecorativeLine:~[0,0.05,0.35]~[0.19999,0.0480388,0.05] bodyId:0 color:~[0.05,0,0
 [0,0,1,0]
 [0,0,0,1]
 
-DecorativeMeshFile:blockMesh.obj  bodyId:1 color:~[0,1,1] indexOnBody:-1 Opacity:1 Rep:2 Scale:~[-1,-1,-1] Transform:
-[1,0,0,0]
-[0,1,0,0]
-[0,0,1,0]
-[0,0,0,1]
```

Comment:
I haven't been following the evolution of this test case too well...how come this block isn't replaced by a similar one for `DecorativeMesh` (instead of `DecorativeMeshFile`? And where is the block for a ContactHalfSpace? Correspondingly, how come it is okay to remove the contact geometry from the `osim` file? 

Is this all tested differently now?